generator-foodq/generators/app2/index.js

```diff
@@ -3,13 +3,17 @@ var _ = require('lodash');
 var path = require('path');
 const Datauri = require('datauri/sync');
 const DEFAULT_IMAGE = require("./images/defaultImage");
+const types = require('@sap-devx/yeoman-ui-types');
 
 module.exports = class extends Generator {
 	constructor(args, opts) {
 		super(args, opts);
 		this.prompts = opts.prompts;
+		this.appWizard = opts.appWizard;
 		this.parentPromptsQuantity = this.prompts.size();
```

Comment:
this.appWizard = types.AppWizard.create(opts);

Reply:
This is incorrect, because you do not know the name of the object passed to a sub generator

Reply:
**object name should always be appWizard!**

https://github.com/SAP/yeoman-ui/pull/391/files#diff-2aa9919e6c081b46db7b1e2ad5d72ecb045858204730797d687e6fac40d1773fL16